src/index.js

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import ReactDOM from 'react-dom/client';
+import { BrowserRouter, Route, Routes } from 'react-router-dom';
+import Categories from './components/Categories';
+import Books from './components/Books';
+import Navbar from './components/Navbar';
+
+const root = ReactDOM.createRoot(document.getElementById('root'));
+root.render(
+  <React.StrictMode>
+    <BrowserRouter>
+      <Navbar />
+      <Routes>
+        <Route path="/" element={<Books />} />
+        <Route path="/categories" element={<Categories />} />
```

Comment:
Has 2 `<Route>`s, one for the BookList component and the other for the Categories component ✔

- [x] BookList component is the default page